app/utils/file_utils.py

```diff
@@ -53,6 +53,26 @@ def load_json_data(file_path: str | Path) -> Any:
     return data
 
 
+def read_text_data(file_path: str | Path) -> Any:
+    """
+    Read the data from given text file path.
+
+    Parameters:
+    -----------
+    file_path: `str | Path`
```

Comment:
use "``" instead of "`" for parameters to maintain the consistency. Modify wherever required.